<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColorCandle</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Qt/projImages/icon.png</normaloff>../../../../Qt/projImages/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="slBrightness">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>30</width>
      <height>220</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>7</number>
    </property>
   </widget>
   <widget class="QSlider" name="slRed">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>30</width>
      <height>220</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>7</number>
    </property>
   </widget>
   <widget class="QSlider" name="slGreen">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>30</width>
      <height>220</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>7</number>
    </property>
   </widget>
   <widget class="QSlider" name="slBlue">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>30</width>
      <height>220</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>7</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelBr">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Brightness</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRed">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>270</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGreen">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBlue">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>270</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RANDOM</string>
    </property>
   </widget>
   <widget class="QLabel" name="lColorView">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>225</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Qt/projImages/icon.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="valueBr">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>255</string>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="valueRed">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>255</string>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="valueGreen">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>255</string>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="valueBlue">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>255</string>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCandleEffect">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Candle Effect : OFF</string>
    </property>
   </widget>
   <widget class="QSlider" name="slBlink">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>230</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>128</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>9</number>
    </property>
   </widget>
   <widget class="QSlider" name="slFade">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>128</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelBlink">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>200</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rainbow Blink</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFade">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>270</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rainbow Fade</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>320</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Volume</string>
    </property>
   </widget>
   <widget class="QSlider" name="vol">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>340</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>128</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>9</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
